.eslintcache

```diff
@@ -1 +1 @@
nt":0,"fixableWarningCount":0,"usedDeprecatedRules":"38"},{"filePath":"45","messages":"46","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"38"},{"filePath":"47","messages":"48","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"49","messages":"50","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"51","messages":"52","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"53","messages":"54","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"38"},{"filePath":"55","messages":"56","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"38"},{"filePath":"57","messages":"58","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"59","messages":"60","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"61","messages":"62","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\37477\\Desktop\\adobe\\src\\index.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\App.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\reportWebVitals.js",[],["63","64"],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\Login\\index.jsx",["65"],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\Login\\styles.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\Button\\index.jsx",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\Button\\styles.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\User\\index.jsx",["66"],"C:\\Users\\37477\\Desktop\\adobe\\src\\UserContext\\index.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\User\\styles.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\LoginField\\index.jsx",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\components\\LoginField\\styles.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\httpClient.js",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\DataList\\index.jsx",[],"C:\\Users\\37477\\Desktop\\adobe\\src\\DataList\\styles.js",[],{"ruleId":"67","replacedBy":"68"},{"ruleId":"69","replacedBy":"70"},{"ruleId":"71","severity":1,"message":"72","line":3,"column":10,"nodeType":"73","messageId":"74","endLine":3,"endColumn":18},{"ruleId":"75","severity":1,"message":"76","line":27,"column":6,"nodeType":"77","endLine":27,"endColumn":16,"suggestions":"78"},"no-native-reassign",["79"],"no-negated-in-lhs",["80"],"no-unused-vars","'Redirect' is defined but never used.","Identifier","unusedVar","react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'history'. Either include it or remove the dependency array.","ArrayExpression",["81"],"no-global-assign","no-unsafe-negation",{"desc":"82","fix":"83"},"Update the dependencies array to be: [getToken, history]",{"range":"84","text":"85"},[825,835],"[getToken, history]"]
```

Comment:
add to gitignore. Its 10 th time im telling you that. Either add in gitignore or not commit this file